WooCommerce/Classes/POS/Presentation/Reusable Views/POSModalManager.swift

```diff
@@ -0,0 +1,19 @@
+import SwiftUI
+
+class POSModalManager: ObservableObject {
+    @Published private(set) var isPresented: Bool = false
+    private var contentBuilder: (() -> AnyView)?
```

Comment:
Not necessarily something that affect us or should be changed, just a note: I recall [reading](https://www.swiftbysundell.com/articles/avoiding-anyview-in-swiftui/) about performance issues due to `AnyView` usage. 

I gave it a couple of tries to attempt to remove the `AnyView` type in favour of using a generic constraint to `View`, but it just seems to make make everything more complicated, since now we have to pass the `<ModalContent: View>` generic types everywhere this is used, so it feels a bit of pre-optimization for no real benefit 😅